website/cue/reference/components/sinks.cue

```diff
@@ -221,6 +256,14 @@ components: sinks: [Name=string]: {
 							}
 						}
 						options: {
+							if codec == "avro" {
```

Comment:
I think what you want here is to check if `features.send.encoding.codec.enum` _contains_ `avro` (see cue failure).